Makefile

```diff
@@ -1,27 +1,26 @@
 include ./make/*.mk
 
 ORG ?= integreatly
-NAMESPACE_PREFIX=redhat-rhmi-
-NAMESPACE=$(NAMESPACE_PREFIX)operator
-PROJECT=integreatly-operator
 REG=quay.io
 SHELL=/bin/bash
-TAG ?= 2.7.0
-MGDAPI_TAG ?= 0.1.0-rc2
+
 PKG=github.com/integr8ly/integreatly-operator
 TEST_DIRS?=$(shell sh -c "find $(TOP_SRC_DIRS) -name \\*_test.go -exec dirname {} \\; | sort | uniq")
 TEST_POD_NAME=integreatly-operator-test
 COMPILE_TARGET=./tmp/_output/bin/$(PROJECT)
 OPERATOR_SDK_VERSION=0.17.1
 AUTH_TOKEN=$(shell curl -sH "Content-Type: application/json" -XPOST https://quay.io/cnr/api/v1/users/login -d '{"user": {"username": "$(QUAY_USERNAME)", "password": "$(QUAY_PASSWORD)"}}' | jq -r '.token')
 TEMPLATE_PATH="$(shell pwd)/templates/monitoring"
-INTEGREATLY_OPERATOR_IMAGE ?= $(REG)/$(ORG)/$(PROJECT):v$(TAG)
-MANAGED_API_OPERATOR_IMAGE ?= $(REG)/$(ORG)/managed-api-service:v$(MGDAPI_TAG)
+
+
 CONTAINER_ENGINE ?= docker
 TEST_RESULTS_DIR ?= "test-results"
 TEMP_SERVICEACCOUNT_NAME="rhmi-operator"
 CLUSTER_URL:=$(shell sh -c "oc cluster-info | grep -Eo 'https?://[-a-zA-Z0-9\.:]*'")
 
+RHMI_TAG ?= 2.7.0
```

Comment:
I see that these get assigned to TAG in the if statement below. Could we just move them down there?

Reply:
These tags are modified by the prepare-release script. In the `if` blocks the TAG is assigned  with the correct value. This value is used in the image url which is also set at this stage. Assigned the TAG value also meant not having to change any existing make targets that maybe using the TAG. Possible I should put in a comment about why the tags are done this way, Phil also asked the question.